src/app/public/modules/media-query/media-query.service.ts

```diff
@@ -65,11 +84,21 @@ export class SkyMediaQueryService implements OnDestroy {
     this.addListeners();
   }
 
+  /**
+   * Angular's ngOnDestroy lifecycle hook.
+   * @internal
```

Comment:
`@internal` not needed for Angular lifecycle hooks (they're removed automatically).

Reply:
Removed.